docs/internet-identity-spec.adoc

```diff
@@ -197,6 +197,8 @@ type DeviceData = record {
   pubkey : DeviceKey;
   alias : text;
   credential_id : opt CredentialId;
+  purpose : variant { recovery; authentication };
+  key_type : variant { unknown; platform; cross_platform; seed_phrase };
```

Comment:
The actual `.did` in the repo has type aliases for these variants. Either is fine.

Reply:
I'll leave it as is. For documentation purposes I think it's easier with less indirections.